documentation/docs/getting_started/rust.mdx

```diff
@@ -28,8 +33,8 @@ version [rustup update stable](https://github.com/rust-lang/rustup.rs#keeping-ru
 The library uses [_cmake_](https://cmake.org/documentation/) and [_openssl_](https://www.openssl.org/docs/). You may
 need install additional build tools on your system to run the build process successfully using Cargo.
 
-### Windows
-
+<Tabs>
+<TabItem value={'windows'} label={'Windows'}>
 You can install all needed prerequisites on Windows with [Chocolatey](https://community.chocolatey.org/packages?q=msys). If you have any of the tools already installed just remove them from the command
```

Comment:
As best practice: If you mix markdown with custom or html elements, always add a new line before and after. Some components might not work otherwise. (In this case the Chocolatey link)

```suggestion
<TabItem value={'windows'} label={'Windows'}>

You can install all needed prerequisites on Windows with [Chocolatey](https://community.chocolatey.org/packages?q=msys). If you have any of the tools already installed just remove them from the command
```